### ISSUE TITLE
[BUG] Quarkus issues not appearing

### ISSUE THREAD

**Issue by jmle** [konveyor/analyzer-lsp#360]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Konveyor version

0.3.0-alpha.5

### Priority

Major

### Current Behavior

No quarkus target issues are appearing in the analysis.

### Expected Behavior

Quarkus issues should appear, see windup results:
![image](https://github.com/konveyor/analyzer-lsp/assets/4527008/4b087b7e-5f04-4c68-8c6d-362bab6cd503)

### How Reproducible

Always (Default)

### Steps To Reproduce

1. In the given environment and configuration, analyze the daytrader application.
2. Go to the issues page and see that no quarkus target issues are appearing

### Environment

```markdown
- Fedora 38
- minikube version: v1.31.2
- Konveyor 0.3.0-alpha.5
- [sample.daytrader7 app (master branch)](https://github.com/WASdev/sample.daytrader7)
- Analysis configuration:
  - Source code + dependencies
  - eap8, quarkus, openjdk17, azure, cloud-readiness, camel3
  - Application and internal dependencies only
  - No custom rules
  - No additional advanced options
```

**Comment by pranavgaikwad** [comment]
This issue comes from a groovy rule in Quarkus.

**Comment by jmle** [comment]
@pranavgaikwad not all issues come from groovy rules, ie "Adopt Maven Compiler plugin" has [an xml rule in windup](https://github.com/windup/windup-rulesets/blob/master/rules/rules-reviewed/quarkus/java-ee/javaee-pom-to-quarkus.windup.xml#L132).

**Comment by pranavgaikwad** [comment]
This is now fixed via https://github.com/konveyor/rulesets/pull/26
